src/FixedPointNumbers.jl

```diff
@@ -40,54 +41,87 @@ export
     # Functions
     scaledual
 
-reinterpret(x::FixedPoint) = x.i
+getindex(x::AbstractFixedPoint) = x.i
```

Comment:
When going through the codebase I found that `reinterpret(x)` and `x.i` where both used, thus defeating the point of using `reinterpret(x)`, also for me personally `reinterpret` makes it harder to read or quickly parse the code. So my main issue with it is that it is to long and it gets in the way. I was thinking about using `get(x)`, but it is semantically as problematic as `getindex`.